annex/restore-test/Vagrantfile

```diff
@@ -0,0 +1,66 @@
+# -*- mode: ruby -*-
+
+# Older versions of vagrant can't start the ubuntu bentobox with
+# private networking
+Vagrant.require_version ">= 1.8.2"
+
+box = "bento/ubuntu-16.04"
+hostname = 'annex-restore-test'
+
+memory = 2048 # in MB
+disk_size = 60 # in GB
+
+ip = '192.168.80.10'
+domain = 'localdomain'
+
+# Pass through github token
+env = { 
+  "VAULT_AUTH_GITHUB_TOKEN" => ENV["VAULT_AUTH_GITHUB_TOKEN"],
+  "VAULT_ADDR" => ENV["VAULT_ADDR"]
+}
+
+Vagrant.configure(2) do |config|
+  # Basic properties
+  config.vm.box = box
+  config.vm.provider :virtualbox do |vbox|
+    vbox.gui = false
+    vbox.memory = memory
+  end
+  config.vm.hostname = "#{hostname}.#{domain}"
+  config.vm.network :private_network, ip: ip
+  config.vm.network "forwarded_port", guest: 15432, host: 15433
+  config.vm.synced_folder 'shared', '/vagrant'
```

Comment:
So this 'shared' folder needs to first be created on the host machine? Maybe add as a step to the README?

Reply:
Sorry, I had thought that was committed. I'd forgotten Git doesn't check in empty folders. Fixed.